### DISCUSSION TITLE
Welcome to Nraphy Discussions!

### DISCUSSION THREAD

**Discussion by RFKaya** [RFKaya/Nraphy#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by c2vs** [comment]
please can you make english versionof nraphy

**Reply by RFKaya** [reply]
I'm planning to do it already, but it will be very difficult to set the whole bot back to English.
